### DISCUSSION TITLE
cut triangulated meshes/surfaces at intersection

### DISCUSSION THREAD

**Discussion by andrea-bistacchi** [cnr-isti-vclab/PyMeshLab#178] [Q&A]
Hello, is it possible to cut a triangulated mesh/surface at the intersection with another one?

I don’t understand if the Boolean tools can be used in this way, particularly on open surfaces.

Thanks very much!

**Comment by alemuntoni** [comment]
Hi, booleans cannot be used with open surfaces: they require closed surfaces in order to exactly discriminate between the interior and the exterior of a surface. Therefore I think there is no way to do what you need using pymeshlab.
